index.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+require_once 'functions.php';
+?>
+
+<!DOCTYPE html>
+<html lang="en-GB">
+
+<head>
+    <title>My Bike Collection</title>
+    <link href="https://fonts.googleapis.com/css?family=M+PLUS+1p|Permanent+Marker&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="Normalize.css">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+    <section>
+        <h1>My Bike Collection</h1>
+
+        <div class="tableHeadings">
+            <div class="pictureHeading">
+                <h2>Picture</h2>
+            </div>
+            <div>
+                <h2>Manufacturer</h2>
+            </div>
+            <div>
+                <h2>Model</h2>
+            </div>
+            <div>
+                <h2>Wheelset</h2>
+            </div>
+            <div>
+                <h2>Groupset</h2>
+            </div>
+        </div>
+
+        <?php
+        $db = dbConn();
```

Comment:
move this procedural code up to the top, just have the echo down here

Reply:
Done